homework-portfolio/homework-portfolio.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="C:\Users\200261\Desktop\FEWD\Week 1\day-2\Assignment-1-Portfolio-Website\homework-portfolio.css">
```

Comment:
@joeyluca2003 don't forget to update your CSS stylesheet when you are pushing to Github!